src/modules/map/components/MapPopover.vue

```diff
@@ -1,24 +1,151 @@
+<script>
+/** @enum */
+export const MapPopoverMode = {
+    FLOATING: 'FLOATING',
+    FEATURE_TOOLTIP: 'FEATURE_TOOLTIP',
+}
+</script>
+<script setup>
+/** Map popover content and styles. Position handling is done in corresponding library components */
+
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import { computed, onMounted, ref, toRefs } from 'vue'
+import { useStore } from 'vuex'
+
+import variables from '@/scss/variables.module.scss'
+import promptUserToPrintHtmlContent from '@/utils/print'
+import { useMovableElement } from '@/utils/useMovableElement.composable'
+import { useTippyTooltip } from '@/utils/useTippyTooltip.js'
+
+const footerHeight = parseInt(variables.footerHeight)
+const devDisclaimerHeight = parseInt(variables.devDisclaimerHeight)
+
+const props = defineProps({
+    authorizePrint: {
+        type: Boolean,
+        default: false,
+    },
+    title: {
+        type: String,
+        default: '',
+    },
+    useContentPadding: {
+        type: Boolean,
+        default: false,
+    },
+    anchorPosition: {
+        type: Object,
+        default: null,
+        validator: (value, props) =>
+            props.mode !== MapPopoverMode.FEATURE_TOOLTIP ||
+            (value && value.top >= 0 && value.left >= 0),
+    },
+    mode: {
+        type: String,
+        default: MapPopoverMode.FLOATING,
+        validator: (value) => Object.values(MapPopoverMode).includes(value),
+    },
+})
+
+const { authorizePrint, title, useContentPadding, anchorPosition, mode } = toRefs(props)
+
+const emits = defineEmits(['close'])
+
+const popoverHeader = ref(null)
+const popover = ref(null)
+
+const showContent = ref(true)
+
+const store = useStore()
+const hasDevSiteWarning = computed(() => store.getters.hasDevSiteWarning)
+const currentHeaderHeight = computed(() => store.state.ui.headerHeight)
+const isPhoneMode = computed(() => store.getters.isPhoneMode)
+
+const cssPositionOnScreen = computed(() => {
+    if (mode.value === MapPopoverMode.FEATURE_TOOLTIP) {
+        return {
+            top: `${anchorPosition.value.top}px`,
+            left: `${anchorPosition.value.left}px`,
+        }
+    }
+    return {}
+})
+
+const popoverLimits = computed(() => {
+    let top = currentHeaderHeight.value
+    if (hasDevSiteWarning.value) {
+        top += devDisclaimerHeight
+    }
+    return {
+        top,
+        bottom: isPhoneMode.value ? 0 : footerHeight,
+        left: 0,
+        right: 0,
+    }
+})
+
+useTippyTooltip('.map-popover-header .print-button[data-tippy-content]')
```

Comment:
Would simply do `useTippyTooltip('.map-popover-header [data-tippy-content]')` and add tippy to other components like the close and collapse buttons